ps5Controller/ps5Controller/HomeModule/HomeViewController.swift

```diff
@@ -37,62 +39,55 @@ extension HomeViewController: HomeViewInterface {
         navigationController?.isNavigationBarHidden = true
         collectionView.dataSource = self
         collectionView.delegate = self
-        collectionView.register(HomeCollectionViewCell.nib(),
+        collectionView.register(HomeCollectionViewCell.nib,
                                 forCellWithReuseIdentifier: HomeCollectionViewCell.identifier)
     }
     
     func reloadData() {
-        DispatchQueue.main.async {
-            self.collectionView.reloadData()
-        }
+        self.collectionView.reloadData()
     }
     
     func animateWhenReloadData() {
-        self.collectionView.transform = CGAffineTransform(translationX: view.frame.width, y: 0)
+        collectionView.transform = CGAffineTransform(translationX: view.frame.width, y: 0).scaledBy(x: 0.6, y: 0.6)
+        collectionView.setContentOffset(CGPoint(x: 0, y: 0), animated: false)
         UIView.animate(withDuration: 1) {
-            self.collectionView.transform = CGAffineTransform(translationX: 0, y: 0)
+            self.collectionView.transform = CGAffineTransform(translationX: 0, y: 0).scaledBy(x: 1, y: 1)
         }
     }
 }
 
 extension HomeViewController: UICollectionViewDataSource {
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        guard let homeViewModels = presenter?.getHomeViewModels() else { return 0 }
-        return homeViewModels.count
+        presenter.numberOfItemsInSection
     }
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: HomeCollectionViewCell.identifier, for: indexPath) as? HomeCollectionViewCell,
-              let homeViewModels = presenter?.getHomeViewModels() else { return UICollectionViewCell() }
-        cell.configure(with: homeViewModels[indexPath.row])
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: HomeCollectionViewCell.identifier, for: indexPath) as! HomeCollectionViewCell
```

Comment:
bi extension yazamaz mısın sadece indexPath ve HomeCollectionViewCell.self type'ını alsın. Döndüğü type verdiğin 
olsun. identifier zaten cell'den alıyosun o extensionınla. tertemiz olur. Bence alttakini yaptırırsın
`let cell = collectionView.dequeueReusableCell(type: HomeCollectionViewCell.self, for: indexPath)`